topics/sequence-analysis/tutorials/annotation-with-prokka/tutorial.md

```diff
@@ -0,0 +1,139 @@
+> (ask your instructor). If this is not the case, you can upload it yourself.
+> - Option 1: From data library:
+>   - Navigate to the shared data library, you should find a file called `contigs.fasta`
+> - Option 2: From your computer:
+>   - Obtain data directly from Zenodo: [![DOI](https://zenodo.org/badge/DOI/10.5281/zenodo.1156405.svg)](https://doi.org/10.5281/zenodo.1156405)
+>   - Download `contigs.fasta`
+>   - Upload the file to your history.
+> <br><br>
+>
+{: .hands_on}
+
+## Annotate the genome
+
+Now we will run the tool called Prokka.
+
+> ### {% icon hands_on %} Hands-on: Annotate genome
+>
+> 1. **Prokka** {% icon tool %} with the following parameters (leave everything else unchanged)
+>    - contigs to annotate: `contigs.fasta`
+>    - Locus tag prefix (--locustag): P
+>    - Force GenBank/ENA/DDJB compliance (--compliant): No
+>    - Sequencing Centre ID (--centre): V
+>    - Genus Name: Staphylococcus  
+>    - Species Name: aureus  
+>    - Use genus-specific BLAST database: No  
+>    - Your tool interface should look like this:
+>    - ![prokka interface](images/interface.png)
+>    - Click Execute
+> <br><br>
+{: .hands_on}
+
+## Examine the output
+
+Once Prokka has finished, examine each of its output files.
+
+> ### {% icon hands_on %} Hands-on: output files
+>
+> - The GFF and GBK files contain all of the information about the features annotated (in different formats.)
+> - The .txt file contains a summary of the number of features annotated.
+> - The .faa file contains the protein sequences of the genes annotated.
+> - The .ffn file contains the nucleotide sequences of the genes annotated.
+> <br><br>
+{: .hands_on}
+
+## View annotated features in JBrowse
+
+Now that we have annotated the draft genome sequence, we would like to view the sequence in the JBrowse genome viewer. First, we have to make a JBrowse file. Then, we can view it within Galaxy.
+
+> ### {% icon hands_on %} Hands-on: Visualize annotation
+>
+> 1. **JBrowse** {% icon tool %} with the following parameters
+> - Under Reference genome to display choose *Use a genome from history*.
+> - Under Select the reference genome choose Prokka on data XX:fna. This .fna sequence is the fasta nucleotide sequence, and will be the reference against which annotations are displayed.
+> - For Produce a Standalone Instance select *Yes*.
+> - For Genetic Code choose *11: The Bacterial, Archaeal and Plant Plastid Code*.
+> - Under JBrowse-in-Galaxy Action choose *New JBrowse Instance*.
+> - Click Insert Track Group
+> - Under Track Category type in *gene annotations*.
+> - Click Insert Annotation Track
+> - For Track Type choose *GFF/GFF3/BED/GBK Features*
+> - For GFF/GFF3/BED Track Data select Prokka on data XX:gff  [Note: not wildtype.gff]
+> - Under JBrowse Track Type[Advanced] select *Canvas Features*.
+> - Click on JBrowse Styling Options <Advanced]
+> - Under JBrowse style.label add in *product*.
```

Comment:
Based on the screenshot, it should be "product,name,id" there

Reply:
Thanks, changed